### DISCUSSION TITLE
Running under Wine / its variations (Proton, Whisky, etc.)

### DISCUSSION THREAD

**Discussion by 0xeb** [OpenSalamander/salamander#53] [General]
I would love, as a quick win, to be able to run OpenSalamander under emulation, but unfortunately, nothing works.

<img width="1406" alt="image" src="https://github.com/OpenSalamander/salamander/assets/230851/d39508c5-5d6b-4116-b808-c1d8095fc745">

For instance, this is Whisky on Mac M1. But I also tried on regular Linux / x86 architecture.

I would love to investigate why.

Any thoughts on that topic?

**Comment by janrysavy** [comment]
OK, I just ordered a MacBook Air M3, I'm tired of waiting for MS and would like the native ARM version of Salamander.

**Reply by 0xeb** [reply]
macOS and Linux would benefit from such a wonderful file manager like Salamander.

But, having surveyed the source code, it looks like very difficult to port to a different UI system since you rely heavily on Windows UI.


Qt would be a best choice if you ever decide to start porting.

--
Elias

**Reply by janrysavy** [reply]
I was thinking primarily of the Windows ARM64 native version.

**Reply by 0xeb** [reply]
FYI:

<img width="721" alt="image" src="https://github.com/OpenSalamander/salamander/assets/230851/84c6fbc5-7b39-4525-bad6-94931e98341b">

But yeah, why not. It should be way more feasible.
